src/crimson/osd/recovery_backend.h

```diff
@@ -135,6 +143,18 @@ class RecoveryBackend {
 	      std::make_error_code(std::errc::interrupted), why));
       }
     }
+    void stop() {
```

Comment:
could you move the implementation into the .cc file so we don't need to include `crimson/common/exception.h` in the header?